evm_body.yaml

```diff
@@ -287,14 +292,20 @@ eth_cancelPrivateTransaction:
 eth_getBalance:
   allOf:
     - $ref: '#/common_request_fields'
-    - $ref: '#/address_and_blockNumber_param_eth'
     - type: object
       properties:
+        address:
+          type: string
+          description: 20 Bytes - Address
+          pattern: '^0[xX][0-9a-fA-F]{40}$'
```

Comment:
since this is the same as above in eth_getTransactionCount we might want to just create a component and reuse it across the board